### ISSUE TITLE
Return colum value as a string regardless of its type

### ISSUE THREAD

**Issue by FungusHumungus** [sfackler/rust-postgres#379]
From what I can see you have to know in advance what type the column is. Sometimes I don't and just want to dump the value to a file.

Is there a way of just fetching a readable string value of the column regardless of its type in postgres?

**Comment by FungusHumungus** [comment]
True, but that is assuming I know what the columns in the table are. Granted I can always find this out with an extra roundtrip to the database, but it would be easier and quicker to just do a `select *`.

**Comment by sfackler** [comment]
You can't cast the columns returned by a `SELECT *` unfortunately.

**Comment by jbg** [comment]
you could consider using Postgres's `row_to_json()` function to retrieve the row without knowing anything about its structure.

**Comment by benesch** [comment]
I believe this is now possible in postgres 0.17 via the `simple_query` function. The returned `SimpleQueryRow` allows you to ask for (only) the text representation of each column: https://docs.rs/postgres/0.17.0/postgres/row/struct.SimpleQueryRow.html
